src/typescript/views/todoView.ts

```diff
@@ -29,6 +33,24 @@ class TodoView {
 
         this.todoListEl.innerHTML += todoTemplate;
     };
+
+    addEventSearch = (): void => {
+        this.btnSearchEl?.addEventListener('click', async (e: MouseEvent) => {
+            e.preventDefault();
+
+            const value: string = this.inputSearchEl.value;
+
+            const nameTodos: NodeList = document.querySelectorAll('.todo-info-name');
+
+            nameTodos.forEach((nameTodo) => {
+                if (nameTodo.textContent?.toLocaleLowerCase().includes(value)) {
+                    (nameTodo.parentNode?.parentNode as HTMLElement).style.display = 'flex';
```

Comment:
Do not style by JS, style should be in CSS

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!